test/image/mocks/table_plain_birds.json

```diff
@@ -0,0 +1,90 @@
+{
+    "layout": {
+        "width": 1000,
+        "height":  1000,
+        "title": "Deep-water horizon oil spill - observed birds<br><sup><i>U.S. Fish & Wildlife Service</i></sup>"
+    },
+
+    "data": [{
+
+        "type": "table",
+
+        "domain": {
+            "x": [0, 1],
+            "y": [0, 1]
+        },
+
+        "columnwidth": [20, 40, 25, 25, 30, 20, 30],
+        "columnorder": [0, 1, 2, 3, 4, 5, 6],
+
+        "header": {
+
+            "height": 30,
+
+            "values": [["<b>#</b>", ""], ["<b>Species</b>", ""], ["Latitude", "$\\phi °$"], ["Longitude", "$\\lambda °$"], ["Oilling", ""], ["Condition", "<i>Live / Dead</i>"], ["Date", "<i>yyyy-mm-dd</i>"]],
+
+            "align": ["right", "left", "right", "right", "left", "left", "left"],
+            "valign": "middle",
+
+            "line": {
+                "color": "lightgray",
+                "width": 0.0
+            },
+
+            "fill": {
+                "color": ["dimgray", "grey"]
+            },
+
+            "font": {
+                "family": "Arial",
+                "size": [[14, 12]],
+                "color": ["white", "white", "white", "white", "white", ["pink", "white"], "white"]
+            }
+        },
+
+        "cells": {
+
+            "values": [
```

Comment:
Hmm. So here `values[i]` represent columns - which I think is fair for `table`. But this means that `cells.values` is the transpose of the `z` attribute in `heatmap` traces.  Which at the very least write this down in the `table/index.js` module description.